x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_actions/management.js

```diff
@@ -162,6 +188,10 @@ export function actionsMenuContent(
   ];
 }
 
+function jobBlocked(item) {
```

Comment:
```suggestion
function isJobBlocked(item) {
```

Reply:
Updated in https://github.com/elastic/kibana/pull/108039/commits/06f04dc840fb5a6cf2d98bd62bf8fa7a77f9b747